Gruntfile.js

```diff
@@ -67,7 +71,13 @@ module.exports = function(grunt) {
           declaration: true,
           module: 'commonjs'
         },
-        src: ['src/tslint.ts'],
+        src: [
+          'typings/*.d.ts',
+          'src/language/walker/syntaxWalker.ts',
```

Comment:
why is syntaxWalker special? if this is for ordering, we need a better way than this.